cmd/manager/main.go

```diff
@@ -69,6 +71,25 @@ func main() {
 
 	printVersion()
 
+	// Checking if required files exist before starting the operator
+	requiredFiles := []string{
+		wkl.FlannelCNIPluginPath,
+		wkl.HostLocalCNIPlugin,
+		wkl.WinBridgeCNIPlugin,
+		wkl.WinOverlayCNIPlugin,
+		wkl.HybridOverlayPath,
+		wkl.KubeletPath,
+		wkl.KubeProxyPath,
+		wkl.IgnoreWgetPowerShellPath,
+		wkl.WmcbPath,
+		wkl.CloudCredentialsPath,
+		wkl.PrivateKeyPath,
+	}
+	if err := checkIfRequiredFilesExist(requiredFiles); err != nil {
+		log.Error(err, "Could not start the operator")
```

Comment:
Should not start with capitals as per the work in #34. `could not...`

Reply:
fixed